### PR TITLE
Add E2e baseline I/O test.

### PR DESCRIPTION
**Pull request by wusui**
Signed-off-by: Warren Usui <wusui@redhat.com>

### CODE REVIEW

File: tests/e2e/test_minimal_io_and_set_baseline.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    )
+    logging.info(
+        f'{io_type}: {io_info.get("clat")} Completion Latency '
+    )
+    with open(f'/tmp/{io_type}_fio_data_baseline.json', 'w') as out_json:
+        json.dump(io_info, out_json)
+
+
+@pytest.mark.usefixtures(
+    create_rbd_secret.__name__,
```
Comment (vasukulkarni) [inline tests/e2e/test_minimal_io_and_set_baseline.py]: This method of fixture is no more valid.

File: tests/e2e/test_minimal_io_and_set_baseline.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+logger = logging.getLogger(__name__)
+
+
+def do_io_actions(io_type, io_info):
+    """
+    Display fio statistics and stash fio results somewhere.
+
+    Args:
+        io_type (str):  either "Read" or "Write"
+        io_info (dict): results of fio run.
```
Comment (vavuthu) [inline tests/e2e/test_minimal_io_and_set_baseline.py]: NIT: Leave empty line
eg:

```
    Args:
        io_type (str):  either "Read" or "Write"
        io_info (dict): results of fio run.

    """
```

Comment (vavuthu) [inline tests/e2e/test_minimal_io_and_set_baseline.py]: io_info (dict) ---> this is the not the complete results of fio run. In the test case, from the test case, we are passing subset of fio results ( like read , write ). Please change the description accordingly.

**Comment by vavuthu** [comment]
@wusui Do we have any threshold values for Read and Write ? In which scenarios this test case will fail.

File: tests/e2e/test_minimal_io_and_set_baseline.py (hunk 1 above)
Comment (wusui) [reply tests/e2e/test_minimal_io_and_set_baseline.py]: How should this be done now that this method is invalid?

File: tests/e2e/test_minimal_io_and_set_baseline.py (hunk 2 above)
Comment (wusui) [reply tests/e2e/test_minimal_io_and_set_baseline.py]: Okay.

Comment (wusui) [reply tests/e2e/test_minimal_io_and_set_baseline.py]: Okay.

**Comment by wusui** [comment]
Vavuthu -- I have sent email to Rick Sussman to get minimum values to check here.  I will add code that will check against these values.

File: tests/e2e/test_minimal_io_and_set_baseline.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    create_rbd_storageclass.__name__,
+    create_project.__name__,
+    create_pvc.__name__,
+    create_rbd_pod.__name__,
+)
+@pytest.mark.polarion_id("OCS-685")
+class TestMinimalLimit(ManageTest):
+
+    @tier1
+    def test_run_io(self):
```
Comment (keesturam) [inline tests/e2e/test_minimal_io_and_set_baseline.py]: This test covers only IO run on rbd volume. please change the function name accordingly. 
We'll also have to cover cephfs and raw block io.

File: tests/e2e/test_minimal_io_and_set_baseline.py (hunk 1 above)
Comment (wusui) [reply tests/e2e/test_minimal_io_and_set_baseline.py]: Has the new method of doing this been implemented yet?

**Comment by RazTamir** [comment]
@wusui see my previous comment - do we still need this?

**Comment by vasukulkarni** [comment]
Closing this one, Please reopen if needed.

**Pull request closed without merging**